python/pynessie/client/nessie_client.py

```diff
@@ -38,7 +38,7 @@
 from ._endpoints import list_tables
 from ._endpoints import merge
 from ..auth import setup_auth
-from ..model import Branch
+from ..model import Branch, ContentKey
```

Comment:
nit: inconsistent import style. why isnt black failing this?

Reply:
True. It seems almost in many places, the import style is mixed up, I am wondering why non of the linter that we have didn't complain about that 🤔. I will look at the linter configs separately

Reply:
Fixed that one